corehq/apps/locations/permissions.py

```diff
@@ -319,6 +319,19 @@ def user_can_access_any_location_id(domain, user, location_ids):
             .exists())
 
 
+def user_can_access_other_user(domain, user, other_user):
+    if user.has_permission(domain, 'access_all_locations'):
+        return True
+
+    accessible_location_ids = (SQLLocation.objects .accessible_to_user(domain, user) .values_list('id', flat=True))
+
+    other_user_locations = other_user.sql_locations
```

Comment:
that seems bad. @sravfeyn was there a reason you returned an empty list instead of an empty queryset? I think we should be returning `SQLLocation.objects.none()` there instead.